Notes.md

```diff
@@ -16,17 +16,6 @@
 ## ReceiptsListViewController
 
 * Better safety nets for dependency injection.
-* Remove the weak from the tableView outlet. Apple recommends strong outlet as of WWDC 2015.
-* All the stuff in viewDidLoad should be broken down into their own self documenting methods.
-	* configureTableView()
-	* loadReceiptsFromDisk()
-	* setupNotificationForAutosave()
-* Add a deinit with a removeObserver call to mirror the setup
-	* `NotificationCenter.default.removeObserver(self)`
-	*  UIViewController will do this for you automatically.
-* Consider removing the auto-row-deselection behavior and putting it in a protocol extension.
```

Comment:
This behavior was moved into the new TableViewManager. I think I'm ok with it being there for now. If we have a larger app with multiple table views I think it might be better to make this behavior into some kind of protocol extension.